### PR TITLE
Copy Edits

### PR DESCRIPTION
**Pull request by tgrubbs61**
Copy edits to multiple pages.

### CODE REVIEW

File: src/app/tool/audit-template/home/home.component.html [hunk 1]
```diff
@@ -1,30 +1,35 @@
...
+  <!-- <div fxFlex="40" fxLayoutGap="20px" fxLayoutAlign="center center">
 		<img src="assets/md/audit-template/img/at-ondemand.svg" alt="cloud and arrows coming out of it">
- 	</div>
+ 	</div> -->
 </div>
-	<p>And the best part is, it’s easy to use.</p>
-	<p>Visit <a href="https://buildingenergyscore.energy.gov/" target="_blank" class="color-1">buildingenergyscore.energy.gov</a> to sign up for an account, or contact [email] to schedule an expert consultation.</p>	
+	<p>Get started by signing up for a free <a href="https://buildingenergyscore.energy.gov/" target="_blank" class="color-1">Audit Template</a> account.</p>
+	<p>To learn more, view the <a href="https://betterbuildingssolutioncenter.energy.gov/webinars/training-introduction-audit-template
+	" target="_blank" class="color-1">Introduction to Audit Template</a> presentation, explore the additional resources on this page or contact BTODataTools@ee.doe.gov.</ think of it as an "easy button" for audits.</p>
```
Comment (kflemin) [inline src/app/tool/audit-template/home/home.component.html:31]:
```suggestion
	" target="_blank" class="color-1">Introduction to Audit Template</a> presentation, explore the additional resources on this page or contact BTODataTools@ee.doe.gov.</ think of it as an "easy button" for audits.</p>
```
```suggestion
	" target="_blank" class="color-1">Introduction to Audit Template</a> presentation, explore the additional resources on this page or contact BTODataTools@ee.doe.gov. Think of it as an "easy button" for audits.</p>
```

File: src/app/tool/bedes/home/home.component.html [hunk 2]
```diff
@@ -1,4 +1,35 @@
...
+    <p>"BEDES Testimonial"</p>
+    <p class="pl-5">— John Doe</p>
+  </div>
+</div> -->
+
+<div fxLayout="row">
+  <div fxFlex="60">
+    <p class="bold">With BEDES, you can easily:</p>
+    <ul>
+      <li>Access, combine, analyze and share building performance data from different sources and formats.
```
Comment (kflemin) [inline src/app/tool/bedes/home/home.component.html:15]:
```suggestion
      <li>Access, combine, analyze and share building performance data from different sources and formats.
```
```suggestion
      <li>Access, combine, analyze and share building performance data from different sources and formats.</li>
```

File: src/app/tool/bedes/home/home.component.html [hunk 3]
```diff
@@ -1,4 +1,35 @@
...
+      <li>Spend less time managing data.</li>
+      <li>Deliver better information to decision makers, partners and the public.</li>
+      <li>Leverage the power of over 30 existing BEDES-compliant tools and datasets.</li>
+      <li>Take the first step on the road to data standardization.</li>
+      <li>Enhance the value of your own products or services.</li>
+      <li>Leverage expert support from DOE and Lawrence Berkeley National Laboratory.</li>
+    </li>
```
Comment (kflemin) [inline src/app/tool/bedes/home/home.component.html:22]:
```suggestion
    </li>
```
```suggestion
```

**Review by kflemin — CHANGES_REQUESTED** [review]
Awesome!  There were just 2 errors that I suggested fixes for (you will have to approve them).  And the quote callout boxes...would you like the background on those to match the color of the section they are in?  (yellow if in developers, green if in analysis & evaluation, etc?)

**Comment by tgrubbs61** [comment]
> 
> 
> Awesome! There were just 2 errors that I suggested fixes for (you will have to approve them). And the quote callout boxes...would you like the background on those to match the color of the section they are in? (yellow if in developers, green if in analysis & evaluation, etc?)

Thanks! I've approved your suggestions, and color-matching the quotes is a good idea. Should I click 'merge pull request'?

**Review by kflemin — APPROVED** [review]

**Pull request merged**